src/schedule/zephyr_domain.c

```diff
@@ -72,10 +72,7 @@ static void zephyr_domain_thread_fn(void *p1, void *p2, void *p3)
 		dt->handler(dt->arg);
 		cycles1 = k_cycle_get_32();
 
-		if (cycles1 > cycles0)
-			diff = cycles1 - cycles0;
-		else
-			diff = UINT32_MAX - cycles0 + cycles1;
+		diff = cycles1 - cycles0;
```

Comment:
I think this is less obvious to the reader, but alas this is simpler (plus there's the of-by-one as the module should be UINT32_MAX+1

Reply:
k_cycle_get_32 returns a value with max32, it is an accumulation of each read based on HW clock(max is 38400000), based on these knowledge, we know that after around 111 seconds, wrap will happens, below logs showed this:
[  111.847735] <inf> ll_schedule: zephyr_domain_thread_fn: ll timer 4294952856
[    0.000626] <inf> ll_schedule: zephyr_domain_thread_fn: ll timer 23858

if multiply 111.847735 or 0.000626 with 38400000, then you will get a similar data with later printed, actually they should same, just because the system debug for timing.

So previous code is correct, simply "-" will have issues when wrap happened.